src/api/calls/getaccount.ts

```diff
@@ -2,8 +2,8 @@
 "use strict";
 
 import { jsonObject, jsonMember } from "typedjson";
-import { ErdstallObject } from "../object";
-import { Address } from "../../ledger";
+import { ErdstallObject } from "#erdstall/api/object";
+import { Address } from "#erdstall/ledger";
```

Comment:
I think this here is a _good_ example for where we want to use package anchor imports: where we have to go into different dirs using `../`s and `../..`s.